src/controllers/places/modify_content.js

```diff
@@ -62,10 +65,13 @@ module.exports = (req, res) => {
   };
   Request(reqOptions, (error, apiResponse, apiResponseBody) => {
     if (error) {
-      res.status(400).send('server error');
+      res.render('error');
```

Comment:
Does this one need some parameters passing to it?

Reply:
@JWLD maybe @Suuuuha so she sees this? Not sure actually if that helps